docs/content/Examples-Tutorials-Recipes/Multiple-Data-Sources-With-Same-Schema.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Manage Access to Multiple Data Sources
+permalink: /recipes/multiple-sources-same-schema
+category: Examples & Tutorials
+subCategory: Data sources
+menuOrder: 3
+---
+
+## Use case
+
+We need to access the data from different data sources for different tenants.
+For example, we are the platform for the online website builder, and each client
+can only view their data. The same data schema is used for all clients.
```

Comment:
> The same data schema is used for all clients.

Do you think it makes sense to show the schema in the "Data Schema" and repeat that it's reused by every tenant?

Reply:
I think just saying this in the text will be enough